x/validator/utils.go

```diff
@@ -10,9 +10,25 @@ import (
 )
 
 func GetCandidateInfoFromMainchain(ctx sdk.Context, keeper Keeper, ethAddress string) (mainchain.CandidateInfo, error) {
```

Comment:
prefer to have two helper functions, instead of wrap DPoSCandidateInfo into CandidateInfo

Reply:
Will update